docs/application/web/guides/multimedia/metadata.md

```diff
@@ -2,34 +2,41 @@
 
 You can extract metadata information from multimedia files.
 
-The main features of the Metadata API include:
-
-- Extraction of a metadata from a file
-  You can [extract metadata](#extract-metadata) from multimedia files using `get()` method.
+The main feature of the Metadata API is extraction of a metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.
 
 
 ## Prerequisites
 
-To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html)), the application has to request [proper storage permissions](../security/privacy-related-permissions.md).
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) and [tv](../../api/latest/device_api/tv/tizen/metadata.html) applications), the application has to request storage privileges by adding the following privileges to the `config.xml` file:
+
+```xml
+<!-- for accessing internal storage only -->
+<tizen:privilege name="http://tizen.org/privilege/mediastorage"/>
+<!-- for accessing external storage only -->
+<tizen:privilege name="http://tizen.org/privilege/externalstorage"/>
+```
+
+Additionally, to access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html) and [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) applications), the application has to request storage privileges [proper storage permissions](../security/privacy-related-permissions.md) using the PPM API.
```

Comment:
@pkosko Do we need this ? this seems repetition. 

As per my understanding :
1. To access files using the Metadata API (in mobile, wearable and tv applications), the application has to request storage privileges by adding the following privileges to the config.xml file:
```
code
```


This seems fine. 

But the next part (line 19) seems to be redundant and only thing that differs is PPM API. Also, you have missed adding TV after wearable in line 19. 

Request you to kindly check, and if you still need line 19, then kindly add TV (If you have missed it).

I would suggest that we can remove it.

Reply:
This information is added on purpose. Storage privileges on mobile and wearable are privacy related, in this case:
1. Need to add privilege to config.xml
2. Use PPM API to `request` privilege from user in runtime (which results with popup asking user to access files). Word 'additionally' was used to show second action which needed.

2nd point is not needed on TV, so I omitted TV on purpose.

Reply:
@pkosko We need slight changes to correct sentence structure. 

Kindly choose the appropriate one and verify for technical accuracy : 
```suggestion 
Additionally, to access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html) and [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) applications), the application has to request [storage privileges](../security/privacy-related-permissions.md) using the PPM API.

```
or 
```suggestion

Additionally, to access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html) and [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) applications), the application has to request [proper storage permissions](../security/privacy-related-permissions.md) using the PPM API.
```